packages/app-check/src/web.ts

```diff
@@ -62,15 +90,34 @@ export class FirebaseAppCheckWeb
   }
 
   public async initialize(options?: InitializeOptions): Promise<void> {
-    if (!options?.siteKey) {
-      throw new Error(FirebaseAppCheckWeb.errorSiteKeyMissing);
+    if (!options) {
+      throw new Error(FirebaseAppCheckWeb.errorInvalidOptions);
```

Comment:
Do we still need this condition?

Reply:
I guess there should always be some options defined by user, but it is a matter of choice here to "fail fast" if `options` are not provided or failing later on when some exact option is missing.

I feel this does the same as the previous `!options?` part.

Reply:
Let's remove it and throw an error as soon as we access an option that is not available to keep the previous error handling.